supervise/main.go

```diff
@@ -78,31 +82,35 @@ MSGID = The message "type". Set to "out" or "err" to specify which stdio
 		fatalf("reporting pid: %v", err)
 	}
 
+	// Dial syslog.
+	conn, err := net.DialUDP("udp", nil, udpAddr)
```

Comment:
curious why this moved before the signal handling. I _think_ DialUDP will never block, but my intuition is to always configure signal handlers as early as possible in case blocking does happen.

Reply:
Ah this is a leftover from when I was using the connection for printing debug messages. I moved the signal handling above even starting the process. The case in which the signal occurs before the process has been started seems to be less important than the case when the process has been started but signal forwarding hasn't been set up.